### ISSUE TITLE
20:26:36: Backslash found where operator expected at (eval 101562) line 1, near "16\" 

### ISSUE THREAD

**Issue by okrick** [DistributedProofreaders/guiguts#475]
**Describe the bug**
I moved the cursor from the end of this string 
“Witness
to the beginning and got this error.  The same thing happened when I repeated it a second time.

21:32:07: Guiguts 1.2.3: If you have any problems, please include any error messages that appear here with your bug report.
20:26:36: Backslash found where operator expected at (eval 101562) line 1, near "16\"
20:26:37: 	(Missing operator before \?)
20:26:37: Use of uninitialized value in concatenation (.) or string at C:/DP/guiguts/lib/Guiguts/SearchReplaceMenu.pm line 819.
20:45:06: Malformed UTF-8 character: \x9c (unexpected continuation byte 0x9c, with no preceding start byte) in subroutine entry at C:/Strawberry/perl/site/lib/Tk.pm line 251.
20:45:06: Malformed UTF-8 character: \x9c (unexpected continuation byte 0x9c, with no preceding start byte) in subroutine entry at C:/Strawberry/perl/site/lib/Tk.pm line 251.
20:45:51: Malformed UTF-8 character: \x9c (unexpected continuation byte 0x9c, with no preceding start byte) in subroutine entry at C:/Strawberry/perl/site/lib/Tk.pm line 251.
20:45:51: Malformed UTF-8 character: \x9c (unexpected continuation byte 0x9c, with no preceding start byte) in subroutine entry at C:/Strawberry/perl/site/lib/Tk.pm line 251.

**To Reproduce**
1. Open Guiguts
2.  Select the end of the string “Witness
3. Press ctrl-left arrow.

**Expected behavior**
I expected the cursor to be between the double quote and the capital W without getting an error message.

**Environment**
* Guiguts version: 1.2.3
* OS and version: Windows 10 Home 20H2
* Perl version: Strawberry Perl (64-bit) 5.30.3.1-64bit
* Perl distribution: Strawberry Perl (64-bit) 5.30.3.1-64bit

**Additional context**
Here is the file:
[noses-src.zip](https://github.com/DistributedProofreaders/guiguts/files/5827951/noses-src.zip)
The 
“Witness 
string is on line 3728.

**Comment by windymilla** [comment]
Looking at the timestamps on the messages, it looks like Guiguts 1.2.3 was started at 21:32, then the first error occurred 11 hours later at 20:26.
This first error is related to when you use things like \L...\E in a regexp - I don't know if you did that (intentionally or not) and possibly mistyped a regex in the search & replace box. I think a \C might have generated that error. Anyway, I think that might actually be a valid error warning you about what you typed.
The next errors were 9 minutes later and are the ones that relate to using Ctrl-arrow to move around.
I have loaded your file into 1.2.3 and tried putting the cursor after Witness and using Ctrl-left-arrow. I didn't get an error - I know this used to go wrong in a previous release, but I thought it was fixed in 1.2.3. Please could you download the file you uploaded here, and see if you can make it go wrong. If you can give me instructions from when you start GG to when you get the error that would be very helpful. I won't be able to fix it without that.
Many thanks

**Comment by okrick** [comment]
I downloaded my zipped file and tried again.  No error messages this time.  

I used a \C \E regex but I think it was Friday.  And that was on a version of this file for testing.  Since the regex worked as expected I doubt I mistyped \L.

I had the same instance of Guiguts running for over a week.  I closed the app last night and reopened it this morning.  Maybe that had something to do with the error disappearing.

You might close this. I presume it can be reopened if something similar recurs.

Thank you,
